rr_computer_vision/src/traffic_light.cpp

```diff
@@ -1,19 +1,156 @@
-/*
- * @file rr_traffic_light
- * @author Adrian Malaran(AdrianMalaran)
- * @author Toni Ogunmade(oluwatoni)
- * @author Jamie Kim
- * @author Jason Leung
- * @competition IARRC 2018
- */
+  //  @file traffic light detection
+  //  @author Yuchi(Allan) Zhao
```

Comment:
Ditto comment from the header file, also make the file tag have the exact same of the file and change the way its commented out to be the same as it was before with the /* and */ for enclosing the comments instead of doing each line individually